### DISCUSSION TITLE
Graph of all degradation

### DISCUSSION THREAD

**Discussion by rob0101** [netzero-labs/netzero#31] [Ideas]
It'd be great to see a graph showing build or install date (X) and percentage degradation (Y) for the entire fleet of Powerwalls you have access to

**Comment by zigam** [comment]
I agree, just have to manage that carefully because Tesla in the past removed some ways of accessing capacity information.

**Reply by rob0101** [reply]
Can't go sharing information that might not be flattering 🤫
